### PR TITLE
Dev/math/add text block styles

### LINKED ISSUE #462: Add Cupertino TextBlock
## What would you like to be added:

Cupertino theme for TextBlock
https://developer.apple.com/design/human-interface-guidelines/ios/controls/labels/
https://api.flutter.dev/flutter/cupertino/CupertinoTextField-class.html

https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/cupertino/text_theme.dart

## For which Platform:

- [x] UWP
- [x] iOS
- [x] Android
- [x] WebAssembly
- [x] macOS

### PR DESCRIPTION
**Pull request by Guidemarcus**
closes #462 

## PR Type

What kind of change does this PR introduce?
- Feature

## Description

Implement textblock styles and textblock and color samples

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Styles/ColorsSamplePage.xaml [hunk 1]
```diff
@@ -289,6 +289,140 @@
...
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<!-- TODO : Replace white foreground with proper StaticResource  -->
+						<ScrollViewer VerticalScrollBarVisibility="Auto">
+							<Grid>
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition x:Name="Column_1"
+													  Width="*" />
```
Comment (andreaSflores18) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Styles/ColorsSamplePage.xaml:341]: Indentation seems off for this file

**Review by andreaSflores18 — APPROVED** [review]

File: src/library/Uno.Cupertino/Uno.Cupertino.csproj [hunk 2]
```diff
@@ -29,6 +29,12 @@
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
 	</ItemGroup>
+	<ItemGroup>
+	  <None Remove="Styles\Controls\TextBlock.xaml" />
+	</ItemGroup>
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Styles\Controls\TextBlock.xaml" />
+	</ItemGroup>
```
Comment (kazo0) [inline src/library/Uno.Cupertino/Uno.Cupertino.csproj:37]: I don't believe this is needed, should be taken care of as part of https://github.com/unoplatform/Uno.Themes/blob/4b8ba72287dc5179b80b76b5bcba5af71427eceb/src/library/Uno.Cupertino/Uno.Cupertino.csproj#L30

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.iOS/Info.plist [hunk 3]
```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDisplayName</key>
 	<string>Uno.Themes.Samples</string>
 	<key>CFBundleIdentifier</key>
-	<string>uno.platform.material</string>
+	<string>uno.platform.themes</string>
```
Comment (kazo0) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.iOS/Info.plist:8]: I don't know why this changed but it needs to stay uno.platform.material for now :(

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.macOS/Info.plist [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleName</key>
 	<string>Uno.Themes.Samples</string>
 	<key>CFBundleIdentifier</key>
-	<string>uno.platform.material</string>
+	<string>uno.platform.themes</string>
```
Comment (kazo0) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.macOS/Info.plist:8]: I don't know why this changed but it needs to stay uno.platform.material for now :(

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Styles/ColorsSamplePage.xaml (hunk 1 above)
Comment (Guidemarcus) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Styles/ColorsSamplePage.xaml:341]: I cannot seem to change the identation on my side. Nothing happens after ctrl+K+D or if I change it manually

**Review by kazo0 — APPROVED** [review]

**Pull request merged**